vector/src/test/java/im/vector/app/features/crypto/quads/SharedSecureStorageViewModelTest.kt

```diff
@@ -102,10 +102,16 @@ class SharedSecureStorageViewModelTest {
             viewModel.handle(SharedSecureStorageAction.UseKey)
 
             test
-                    .assertState(aViewState(
```

Comment:
`assertState` was removed in favour of a helper to assert all the states (including the initial state)